src/main/java/picard/analysis/artifacts/ContextAccumulator.java

```diff
@@ -178,7 +184,7 @@ private void countRecord(final SAMRecord rec) {
             }
         }
 
-        private void includeCounts(final AlignmentAccumulator other){
+        private void mergeCounts(final AlignmentAccumulator other) {
```

Comment:
Given that this class is nothing but counts, I'd call this method `merge`, not `mergeCounts`